quilt/tools/build.py

```diff
@@ -149,10 +160,7 @@ def _generate_contents(dir_path):
                 nodename, ext = splitext_no_dot(name)
                 ext = ext.lower()
                 rel_path = os.path.relpath(path, startpath)
-                if ext in TARGET['pandas']:
-                    data = [ext, rel_path]
-                else:
-                    data = ['raw', rel_path]
+                data = dict(file=rel_path)
```

Comment:
Including the transform as a key here will (I think) make it clearer to users what build is doing to create the package and (hopefully) make it clearer how they can modify the build settings.